config/tokens.ts

```diff
@@ -121,6 +121,13 @@ const config: {
       dataStreamFeedDecimals: 18,
       oracleTimestampAdjustment: 1,
     },
+    EIGEN: {
+      synthetic: true,
```

Comment:
i don't think we should use synthetic for this, since the token is available on Arbitrum

https://x.com/CamelotDEX/status/1841088733560320393

https://arbiscan.io/address/0x606c3e5075e5555e79aa15f1e9facb776f96c248